src/components/PlantingSites/PlantingSitesList.tsx

```diff
@@ -39,20 +53,47 @@ export default function PlantingSitesList(props: PlantingSitesListProps): JSX.El
   return (
     <TfMain>
       <PageHeaderWrapper nextElement={contentRef.current}>
-        <Box sx={{ paddingBottom: theme.spacing(3) }}>
+        <Box sx={{ paddingBottom: theme.spacing(3), display: 'flex', justifyContent: 'space-between' }}>
           <Grid item xs={6}>
             <Typography fontSize='24px' fontWeight={600}>
               {strings.PLANTING_SITES}
             </Typography>
           </Grid>
+          {searchResults && searchResults.length > 0 ? (
+            <Grid item xs={6} sx={{ textAlign: 'right' }}>
+              {isMobile ? (
+                <Button
+                  id='new-planting-site'
+                  icon='plus'
+                  onClick={() => goTo(APP_PATHS.PLANTING_SITES_NEW)}
+                  size='medium'
+                />
+              ) : (
+                <Button
+                  id='new-planting-site'
+                  icon='plus'
+                  label={strings.ADD_PLANTING_SITE}
+                  onClick={() => goTo(APP_PATHS.PLANTING_SITES_NEW)}
+                  size='medium'
+                />
+              )}
+            </Grid>
+          ) : null}
         </Box>
       </PageHeaderWrapper>
       <Grid>
         <Box>
-          {searchResults?.length === 0 && (
+          {searchResults?.length === 0 ? (
             <Container sx={{ paddingY: 4 }}>
               <EmptyStatePage pageName={'PlantingSites'} />
             </Container>
+          ) : (
+            <PlantigSitesTable
```

Comment:
this is flashing before search finishes, we may need to hide this while search is happening